src/kernel.cr

```diff
@@ -73,7 +73,7 @@ end
 # The syntax for a format specifier is:
 #
 # ```text
-# %[flags][width][.precision]type
+# %[flags][width][.precision][type]
```

Comment:
What? The type is required. That's why it is not in brackets.

Reply:
Oh I didn't realize the brackets indicate that the argument is optional.